js/nullish-coalescing-is-not-as-scary-as-it-sounds/nullish-coalescing-is-not-as-scary-as-it-sounds.md

```diff
@@ -0,0 +1,50 @@
+Nullish Coalescing Operator( **??** ) is one of the features of **ES2020**. It gives the ability to truly check the **nullish** values. It is another logical operator other than OR ( || ) , AND ( && ) and NOT ( ! ) operators.
+
+####**How it works?**
+
+The operator returns its **Right Hand Side (RHS)** operand when its **Left Hand Side (LHS)** operand is **null** or **undefined**, otherwise it will return the LHS operand.
+
+Let's see how it works through some examples:
+
+![Nullish-Coalescing-Operator-Ex-1](https://res.cloudinary.com/dyyr6kwla/image/upload/v1590838686/nullish-coalescing/nullish-example-1_xaqjn0.png)
+
+In the above example, the component Search has a prop [[searchBoxPlaceholder]].
+As the prop is an optional one, so the component uses a default value when the prop is undefined. This is a best use-case of Nullish Coalescing operator because the developer doesn't wants to restrict component to always have a placeholder for the search input box. So using this an empty string can also be passed for the prop and the component will not use the default value for placeholder and would instead do not show a placeholder string, which can not be accomplished using the logical OR operator.
+
+Below are few other examples, which will give idea about more of it's use-cases.
+
+![Few-Other-Examples](https://res.cloudinary.com/dyyr6kwla/image/upload/v1590839423/nullish-coalescing/few-other-examples_ohz48e.png)
+
+####**Comparison with other similar operators**
+
+The operator is simiar to logical **OR** operator and **Default Assignment** opertaor. If you would have noticed in the first example, then we are just assigning a default value to the prop [[searchBoxPlaceholder]], which can also be done in following way:
```

Comment:
similar, operator (spell check)
no need of "then" here
just go with "we are just..."